packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -668,26 +668,93 @@ export function createLodestarMetrics(
         help: "Time elapsed between block slot time and the time block processed",
         buckets: [0.5, 1, 2, 4, 6, 12],
       }),
-      receivedToGossipValidate: register.histogram({
-        name: "lodestar_gossip_block_received_to_gossip_validate",
-        help: "Time elapsed between block received and block validated",
-        buckets: [0.05, 0.1, 0.3, 0.5, 0.7, 1, 1.3, 1.6, 2, 2.5, 3, 3.5, 4],
-      }),
-      receivedToStateTransition: register.histogram({
-        name: "lodestar_gossip_block_received_to_state_transition",
-        help: "Time elapsed between block received and block state transition",
-        buckets: [0.05, 0.1, 0.3, 0.5, 0.7, 1, 1.3, 1.6, 2, 2.5, 3, 3.5, 4],
-      }),
-      receivedToSignaturesVerification: register.histogram({
-        name: "lodestar_gossip_block_received_to_signatures_verification",
-        help: "Time elapsed between block received and block signatures verification",
-        buckets: [0.05, 0.1, 0.3, 0.5, 0.7, 1, 1.3, 1.6, 2, 2.5, 3, 3.5, 4],
-      }),
-      receivedToExecutionPayloadVerification: register.histogram({
-        name: "lodestar_gossip_block_received_to_execution_payload_verification",
-        help: "Time elapsed between block received and execution payload verification",
-        buckets: [0.05, 0.1, 0.3, 0.5, 0.7, 1, 1.3, 1.6, 2, 2.5, 3, 3.5, 4],
-      }),
+
+      gossipValidation: {
+        recvToValLatency: register.histogram({
+          name: "lodestar_gossip_block_received_to_gossip_validate_latency",
+          help: "Time elapsed between block received and gossip validation start",
+          buckets: [0.05, 0.1, 0.3, 0.5, 0.7, 1, 1.3, 1.6, 2, 2.5, 3, 3.5, 4],
+        }),
```

Comment:
We don't need all three here. This one can be removed and just be computed by  `lodestar_gossip_block_received_to_gossip_validate - lodestar_gossip_block_gossip_validate_time`

Reply:
agreed, but its just straight forward, lemme know if you strongly feel about it, can remove

Reply:
I tend to agree with @wemeetagain here.  And same goes for each of these groups `stateTransition`, `signatureVerification`, `executionPayload`, `blockImport`, `gossipBlob`.  Just removing the one extra histogram for the 6 (with 13 buckets each) will reduce total metrics points by 78.  And it can easily be recreated on the dashboard